lib/src/collection/treeset.dart

```diff
@@ -438,7 +436,7 @@ class AvlTreeSet<V> extends TreeSet<V> {
   }
 
   bool remove(Object item) {
-    AvlNode<V> x = _getNode(item);
+    AvlNode<V> x = _getNode(item as V);
```

Comment:
`_getNode` feeds `item` into `comparator`, which is generic and demands a `V`. I would therefore go further and make `item` parameter `V` rather than `Object`.

@cbracken What do you think?